docs/source/docs/examples/aimingatatarget.md

```diff
@@ -1,16 +1,16 @@
 # Aiming at a Target
 
-The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)/[C++](https://github.com/PhotonVision/photonvision/tree/master/photonlib-cpp-examples/aimattarget)).
+The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)).
 
 ## Knowledge and Equipment Needed
 
 - Robot with a vision system running PhotonVision
-- Target
-- Ability to track a target by properly tuning a pipeline
+- AprilTag 7
```

Comment:
Clarify that this is apriltag with ID 7 (looked like typo at first to me) and maybe hyperlink back to where users can find these resources before they run off.